Project.toml

```diff
@@ -7,6 +7,7 @@ version = "0.1.0-pre"
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 GPUArraysCore = "46192b85-c4d5-4398-a991-12ede77f4527"
+HOHQMesh = "e4f4c7b8-17cb-445a-93c5-f69190ed6c8c"
```

Comment:
`HOHQMesh` is only used in the examples and not Raven.  I suggest we move this to `examples/grids/Project.toml`